test/secretbuffer.jl

```diff
@@ -80,4 +80,15 @@ using Test
         @test read(sb, String) == "\xff\xff\xff"
         shred!(sb)
     end
+    @testset "testing the skip function" begin
+    sb = SecretBuffer("computer")
```

Comment:
Indentation.

Reply:
Sorry about that! Changed it in the new commit!